bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
```

Comment:
and keep `require(path.resolve('./runfile'))` out of try/catch blocks, because that will raise silent/false errors reporting